tests/test_pipeline/test_pipeline.py

```diff
@@ -23,11 +23,38 @@
 from etna.pipeline import Pipeline
 from etna.transforms import AddConstTransform
 from etna.transforms import DateFlagsTransform
+from etna.transforms import FilterFeaturesTransform
+from etna.transforms import LogTransform
 from tests.utils import DummyMetric
 
 DEFAULT_METRICS = [MAE(mode=MetricAggregationMode.per_segment)]
 
 
+@pytest.fixture
+def sinusoid_ts():
+    periods = 100
+    sinusoid_ts_1 = pd.DataFrame(
```

Comment:
We have a bunch of methods for artificial datasets generation in `etna.datasets.datasets_generation.py`, try to use for example `generate_ar_df` to generate dataset(to generate "feature_1" you can also use it, just rename the column) 
This methods might be useful when you need to create a random dataset